oak/server/rust/oak_runtime/src/runtime/tests.rs

```diff
@@ -43,11 +43,18 @@ fn run_node_body(node_label: &Label, node_body: Box<NodeBody>) {
 
     struct TestNode {
         node_body: Box<NodeBody>,
+        done: Arc<(Mutex<bool>, Condvar)>,
```

Comment:
Could this test be un-ignored now?

https://github.com/project-oak/oak/blob/11f96d62555364286881e3cc081c98d5754c5f19/oak/server/rust/oak_runtime/src/runtime/tests.rs#L71-L84

Reply:
No, because the test framework is only watching for/catching panics on the current thread &ndash; it doesn't seem to catch panics that occur on a separate thread that was started by the code under test.